### DISCUSSION TITLE
Multiple errors in console with the pinia use tab.

### DISCUSSION THREAD

**Discussion by inane** [vuejs/devtools-v6#2101] [Q&A]
Hello,

From a long time I am having problems with the extension, vue3, typescript and pinia. In the console appears multiple errors with this message. Errors appears when I click in the pinia tab,it is like in that moment pinia feature in the dev tools is activated.

<img width="923" alt="Captura de pantalla 2023-09-08 a las 12 40 34" src="https://github.com/vuejs/devtools/assets/1516015/a0f62e22-462c-49da-aa25-7fcf3ebbb521">

And the use is very tedious. 

Any with this problem

Thank you

**Comment by Azurewarth0920** [comment]
@inane 
Can you provide a minimum reproduction?
